### PR TITLE
Init

### PR DESCRIPTION
**Pull request by smiller171**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+[![yamllint](https://github.com/rhythmictech/terraform-aws-imagebuilder-component-shell/workflows/yamllint/badge.svg?branch=main&event=push)](https://github.com/rhythmictech/terraform-aws-imagebuilder-component-shell/actions?query=workflow%3Ayamllint+event%3Apush+branch%3Amain)
+[![misspell](https://github.com/rhythmictech/terraform-aws-imagebuilder-component-shell/workflows/misspell/badge.svg?branch=main&event=push)](https://github.com/rhythmictech/terraform-aws-imagebuilder-component-shell/actions?query=workflow%3Amisspell+event%3Apush+branch%3Amain)
+[![pre-commit-check](https://github.com/rhythmictech/terraform-aws-imagebuilder-component-shell/workflows/pre-commit-check/badge.svg?branch=main&event=push)](https://github.com/rhythmictech/terraform-aws-imagebuilder-component-shell/actions?query=workflow%3Apre-commit-check+event%3Apush+branch%3Amain)
+<a href="https://twitter.com/intent/follow?screen_name=RhythmicTech"><img src="https://img.shields.io/twitter/follow/RhythmicTech?style=social&logo=RhythmicTech" alt="follow on Twitter"></a>
```
Comment (sblack4) [inline README.md]:
```suggestion
<a href="https://twitter.com/intent/follow?screen_name=RhythmicTech"><img src="https://img.shields.io/twitter/follow/RhythmicTech?style=social&logo=twitter" alt="follow on Twitter"></a>
```

**Review by sblack4 — APPROVED** [review]
:shipit:

**Review by sblack4 — APPROVED** [review]
:shipit:

**Comment by smiller171** [comment]
@sblack4 ran into some issues these new commits solve

File: outputs.tf [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+data "aws_region" "current" {}
+
+locals {
+  account_id = data.aws_caller_identity.current.account_id
+  region     = data.aws_region.current.name
+}
+
+output "component_arn" {
+  description = "ARN of the EC2 Image Builder Component"
+  value       = "arn:aws:imagebuilder:${local.region}:${local.account_id}:component/${lower(var.name)}/${var.component_version}/1"
```
Comment (sblack4) [inline outputs.tf:11]: Again, doesn't the name have a uuid affixed to it?

Comment (smiller171) [reply outputs.tf:11]: only the name of the cloudformation stack, not the name of the component

**Pull request merged**